rotkehlchen/db/dbhandler.py

```diff
@@ -631,35 +631,40 @@ def set_settings(self, write_cursor: 'DBCursor', settings: ModifiableDBSettings)
         CachedSettings().update_entries(settings)
 
     @overload
-    def get_cache(self, cursor: 'DBCursor', name: DBCache) -> Timestamp:
+    def get_cache(self, cursor: 'DBCursor', name: DBCache | str) -> Timestamp | None:
         ...
 
     @overload
-    def get_cache(self, cursor: 'DBCursor') -> dict[DBCache, Timestamp]:
+    def get_cache(self, cursor: 'DBCursor') -> dict[DBCache | str, Timestamp]:
         ...
 
     def get_cache(
             self,
             cursor: 'DBCursor',
-            name: DBCache | None = None,
-    ) -> dict[DBCache, Timestamp] | Timestamp:
+            name: DBCache | str | None = None,
+    ) -> dict[DBCache | str, Timestamp] | Timestamp | None:
         """Returns the cache values from the `key_value_cache` table of the DB
         - If `name` is not passed, it returns all the values in the table
-        - If `name` is passed, it returns the value of the given name, defaults to Timestamp(0)"""
+        - If `name` is passed, it returns the value of the given name, defaults to None"""
         if name is None:
             cursor.execute('SELECT name, value FROM key_value_cache;')
             return {
-                DBCache.deserialize(q[0]): Timestamp(int(q[1]))
+                maybe_get_cache_enum(q[0]): Timestamp(int(q[1]))
                 for q in cursor
             }
```

Comment:
as discussed I believe you can remove this from here and add a method to query only the values that you need to render in the api response or use something that you already have ( I believe you already had something jejeejej)